backend/azure/options_test.go

```diff
@@ -0,0 +1,81 @@
+package azure
+
+import (
+	"encoding/base64"
+	"os"
+	"testing"
+
+	"github.com/Azure/azure-storage-blob-go/azblob"
+	"github.com/stretchr/testify/suite"
+)
+
+type OptionsTestSuite struct {
+	suite.Suite
+}
+
+func (s *OptionsTestSuite) TearDownTest() {
+	clearEnv()
+}
+
+func (s *OptionsTestSuite) TestNewOptions() {
+	o := NewOptions()
+	s.NotNil(o, "when NewOptions returns an error we expect to get a nill options struct")
+}
+
+func (s *OptionsTestSuite) TestCredentials_ServiceAccount() {
+	options := Options{
+		AccountName:            "foo",
+		TenantID:               "foo",
+		ClientID:               "foo",
+		ClientSecret:           "foo",
```

Comment:
Do we know what these values would be expected to be? To test that the numbers of the ID etc will parse correctly?

Reply:
All we know is that the values need to be valid credentials.  The credentials are validated on authentication.